client/layout/index.jsx

```diff
@@ -40,16 +40,9 @@ Layout = React.createClass( {
 
 	_sitesPoller: null,
 
-	componentWillUpdate: function( nextProps, nextState ) {
+	componentWillUpdate: function( nextProps ) {
 		if ( this.props.section !== nextProps.section ) {
-			if ( nextProps.section === 'sites' ) {
-				setTimeout( function() {
-					if ( ! this.isMounted() || this._sitesPoller ) {
-						return;
-					}
-					this._sitesPoller = PollerPool.add( this.props.sites, 'fetchAvailableUpdates', { interval: 900000 } );
-				}.bind( this ), 0 );
-			} else {
+			if ( nextProps.section !== 'sites' ) {
 				this.removeSitesPoller();
```

Comment:
Yeah, I'm not saying remove the poller component from the project, I mean not requiring it in `/layout/index.jsx`! We are not using it anymore there ... or is there something I'm not catching?